### PR TITLE
i18n(fr): Updating `astro-components` translation

### PR DESCRIPTION
**Pull request by Chrissdroid**
#### What kind of changes does this PR include?

- [x] New or updated content
- [x] Translated content

#### Description

- What does this PR change? Updates content from `src/pages/fr/core-concepts/astro-components.md` to reflect latest changes

### CODE REVIEW

File: src/pages/fr/core-concepts/astro-components.md [hunk 1]
```diff
@@ -148,9 +150,17 @@ const items = ["Chien", "Chat", "Ornithorynque"];
...
-Par contre, quand vous utilisez une expression JSX pour créer dynamiquement plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation de `<Fragment> </Fragment>` ou des `<> </>`.
+```astro
+---
+// Template avec plusieurs éléments
+---
+<p>Pas besoin d'entourer les éléments avec un élément conteneur.</p>
+<p>Astro supporte plusieurs éléments racines dans son Template.</p>
+```
+
+Par contre, quand vous utilisez une expression pour créer plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation de `<Fragment> </Fragment>` ou des `<> </>`.
```
Comment (delucis) [inline src/pages/fr/core-concepts/astro-components.md:163]: Noticing “de `<Fragment>`” and “des `<>`”. Should they match?

**Review by delucis — COMMENTED** [review]
Thanks @Chrissdroid! I have one query but otherwise LGTM 🚀

File: src/pages/fr/core-concepts/astro-components.md (hunk 1 above)
Comment (Chrissdroid) [reply src/pages/fr/core-concepts/astro-components.md:163]: i think it should be the same, i just forget one "s", oops

```suggestion
Par contre, quand vous utilisez une expression pour créer plusieurs éléments, vous devez entourer ces éléments d'un **Fragment** comme vous le feriez dans du JavaScript ou du JSX. Astro permet l'utilisation des `<Fragment> </Fragment>` ou des `<> </>`.
```

**Comment by Yan-Thomas** [comment]
Thanks @Chrissdroid! Time to merge 🚀

**Comment by Chrissdroid** [comment]
I've updated the file with the new content change, TYSM @Yan-Thomas for the notification ❤️ 

I'll ask for another review on this change on the discord and hope someone will take the time on this ! 👍

**Review by delucis — APPROVED** [review]
LGTM! Thanks @Chrissdroid 🚀

**Pull request merged**